autokeras/hypermodel/preprocessor.py

```diff
@@ -141,21 +171,29 @@ def output_types(self):
     def output_shape(self):
         return self.inputs[0].shape
 
-    def get_state(self):
+    def get_weights(self):
         return {'sum': self.sum,
                 'square_sum': self.square_sum,
                 'count': self.count,
                 'mean': self.mean,
                 'std': self.std,
                 '_shape': self._shape}
 
-    def set_state(self, state):
-        self.sum = state['sum']
-        self.square_sum = state['square_sum']
-        self.count = state['count']
-        self.mean = state['mean']
-        self.std = state['std']
-        self._shape = state['_shape']
+    def set_weights(self, weights):
+        self.sum = weights['sum']
+        self.square_sum = weights['square_sum']
+        self.count = weights['count']
+        self.mean = weights['mean']
+        self.std = weights['std']
+        self._shape = weights['_shape']
```

Comment:
What is `_shape`? Note that state elements that end up in saved files should generally not be underscore-private.